sorc/link_workflow.sh

```diff
@@ -68,7 +100,11 @@ cd ${pwd}/../parm               ||exit 8
     $LINK ../sorc/gldas.fd/parm                              gldas
 cd ${pwd}/../scripts            ||exit 8
     $LINK ../sorc/gfs_post.fd/scripts/exgdas_atmos_nceppost.sh .
-    $LINK ../sorc/gfs_post.fd/scripts/exgfs_atmos_nceppost.sh  .
+    if [ $model = "coupled" ]; then
+      $LINK exgfs_nceppost_cpl.sh exgfs_atmos_nceppost.sh
```

Comment:
This seems to be the last special section for "coupled", I forget what the issue/difference is here, but should we open an issue(s) somewhere to get this resolved so we don't have to have these two different copies?

Reply:
Well, there's one other "coupled" section, but comment still remains.

Reply:
I expect there will be a bunch of little things like this to hammer out as we determine which is because newer UFS and which is because coupled.